single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultDiagramStyleProvider.java

```diff
@@ -86,12 +86,12 @@ public String getIdWireStyle(Edge edge) {
     }
 
     @Override
-    public Optional<String> getNode3WTStyle(Fictitious3WTNode node, boolean rotate, String vId, String idWinding) {
+    public Optional<String> getNode3WTStyle(Fictitious3WTNode node, boolean rotate, String vId, String nameSubComponent) {
```

Comment:
One of the goal of this issue is to get rid of some multi part component like 3wt. Is it possible to remove getNode3WTStyle and getNode2WTStyle and only use getNodeStyle?

Reply:
A unique method getNodeSVGStyle is now defined in DiagramStyleProvider, to get the style of a SVG component node.